packages/wallet/src/redux/selectors.ts

```diff
@@ -20,16 +19,6 @@ export const getChannelState = (state: SharedData, channelId: string): ChannelSt
   return channelStatus;
 };
 
-// TODO: Ideally we should be able to pass in a expected state type and have the selector either return that type of state or throw an error
-export function getIndirectFundingState(
```

Comment:
Why was this removed?

Reply:
It's been obsoleted now that the reducers are handling `protocolState` directly. It should have been removed earlier but I just caught it now.